projects/plugins/jetpack/class.jetpack.php

```diff
@@ -2768,6 +2761,31 @@ public static function get_module( $module ) {
 		return apply_filters( 'jetpack_get_module', $mod, $module, $file );
 	}
 
+	/**
+	 * Returns whether the file associated with the given slug has no module info.
+	 *
+	 * @param string $slug The slug name.
+	 *
+	 * @return bool Whether the file has no module info.
+	 */
+	private static function has_no_module_info( $slug ) {
```

Comment:
Should this function be generated in `modules/module-headings.php` too? Or should we just add the module info to the files?